tests/cli_tests/test_dsn.py

```diff
@@ -30,6 +33,9 @@ def test_using_dsn():
     assert cli.status == 1
 
 
+@pytest.mark.skipif(
+    not os.path.exists("/tmp/redis/redis.sock"), reason="unix socket is not found"
+)
```

Comment:
That's really wired. I didn't even know how does this test pass before. It should fail on master's build, but it passed.

:(

I tried to run it localy, if I run them all `pytest` this will pass, but if I only run `pytest tests/cli_tests/test_dsn.py` this will fail.

Reply:
OK I see, it's because the previous test set the environment `IREDIS_URL`, so in this test it actually connected via URL instead of socket.

Test socket in github action is hard, I still didn't find a way to test, currently skip it is ok for me.

Thanks